includes/class-sensei-question.php

```diff
@@ -741,14 +761,15 @@ public static function answer_feedback_notes( $question_id ) {
 				<div class="sensei-message info info-special answer-feedback">
 
 					<?php
-
 						/**
-						 * Filter the answer feedback
-						 * Since 1.9.0
+						 * Filter the answer feedback.
 						 *
-						 * @param string $answer_notes
-						 * @param string $question_id
-						 * @param string $lesson_id
+						 * @since 1.9.0
+						 * @hook sensei_question_answer_notes
+						 *
+						 * @param {bool|string} $answer_notes Answer notes.
+						 * @param {int}         $question_id  Question ID.
+						 * @param {bool|int}    $lesson_id    Lesson ID.
```

Comment:
Same here `$lesson_id` and `$answer_notes` can't be a bool at this point.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/3569/commits/38f28a29416247db0646b5fb02cf51ec03de38f3 and https://github.com/Automattic/sensei/pull/3569/commits/5efd7a4cc770f2956f43c1394474cea8564b9d93.